xapian-applications/omega/templates/query

```diff
@@ -49,17 +50,18 @@ document.write("<span title=\""+D+" "+T+"\">Modified:<br><b>"+D+"</b></span>");
 // -->
 </script>
 </head>
+
 <body bgcolor="#ffffff">
```

Comment:
Probably that `bgcolor` should go, as that's really presentational (also it's not great to set the background colour but not the text colour and modern browsers default to white anyway).